docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/documentManagementOnTemplate/addDocument/fields/formula/Formula.rst

```diff
@@ -0,0 +1,146 @@
+=============
+Formula Field
+=============
+
+.. toctree::
+
+Dynamic Formula field has been added to the template page.
+There are three functions available in the formula box:
+
+1. SUM - argument summation
+2. PRODUCT - multiplication of arguments
+3. SUM table column - summing arguments in a dynamic table column
+4. SUBTRACT - returns the difference of arguments
+5. DIVIDE - returns the quotient of arguments
+6. COUNTA table column - returns the number of column cells in a table field
+7. MIN table column - returns the smallest number in a set of arguments
+8. MAX table column - returns the largest value in a set of arguments
+
+arguments can be: numeric, currency, dictionary field, lookup, duplicate, formula field and also the user can enter his own value
+
+Create Formula Field
+====================
+
+1. Create Template
+
+2. Go to template create number field and currency field
+
+.. image:: pic_formula/Screenshot_1.png
+   :width: 600
+   :align: center
+
+3. Select formula field
+
+.. image:: pic_formula/Screenshot_11.png
+   :width: 600
+   :align: center
+
+4. Select the SUM function, in the arguments field, select a numeric and currency fields
+
+.. image:: pic_formula/Screenshot_12.png
+   :width: 600
+   :align: center
+
+4. Save the formula field. Fill in the main fields of  template. Create a draft envelope
+
+5. Filling numeric and currency field
+
+.. image:: pic_formula/Screenshot_3.png
+   :width: 600
+   :align: center
+
+6.After filling fields, pay attention to  formula field
+
+.. image:: pic_formula/Screenshot_4.png
+   :width: 600
+   :align: center
+
+7. Send envelope
+
+In order to apply the SUM table column function, you must repeat the following steps:
+
+1. Create Template
+
+2. Go to template create  dynamic table, added numeric field to dynamic table
```

Comment:
divide actions with comas

"added numeric..." is not correct, you have to use "add numeric..." instead